sdlf-utils/pipeline-examples/datalake-workload-management/lambda/rerun/src/lambda_function.py

```diff
@@ -0,0 +1,77 @@
+import json, boto3, datetime
+import logging
+import os
+import urllib.parse
+from botocore.exceptions import ClientError
+import uuid
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+HIGH = os.environ['HIGH']
+LOW = os.environ['LOW']
+ddb = os.environ['ddb'] 
+
+dynamodb_resource = boto3.resource('dynamodb')
+pipeline_table = dynamodb_resource.Table(ddb)
+sqs_resource = boto3.resource('sqs')
```

Comment:
same as above

Reply:
Changed the way repository is structured -- Now contain two sections. sdlf-wlm-integration now uses interfaces and configurations